applications/MappingApplication/custom_utilities/interface_node.h

```diff
@@ -83,60 +80,103 @@ namespace Kratos
       ///@name Operations
       ///@{
 
-      int GetObjectId() override {
```

Comment:
Usually we use the Allman convection https://en.wikipedia.org/wiki/Indent_style#Allman_style, which means we indent the brackets:

~~~c
while (x == y)
{
    something();
    somethingelse();
}
~~~